automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -155,9 +155,25 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
     }
   }
 
-  // This testcase requires pulling in new code from develop branch
   "import a method config from a workspace" in withWebDriver { implicit driver =>
+    val user = UserPool.chooseProjectOwner
+    implicit val authToken: AuthToken = user.makeAuthToken()
+    withWorkspace(billingProject, "Test_copy_method_config_from_workspace_src") { sourceWorkspaceName =>
+      withWorkspace(billingProject, "Test_copy_method_config_from_workspace_dest") { destWorkspaceName =>
+        val method = MethodData.SimpleMethod
+        api.methodConfigurations.createMethodConfigInWorkspace(billingProject, sourceWorkspaceName,
+          method, method.methodNamespace, method.methodName, 1, Map.empty, Map.empty, method.rootEntityType)
+
+        withSignIn(user) { listPage =>
+          val methodConfigTab = listPage.enterWorkspace(billingProject, destWorkspaceName).goToMethodConfigTab()
+
+          val methodConfigDetailsPage = methodConfigTab.copyMethodConfigFromWorkspace(
+            billingProject, sourceWorkspaceName, method.methodNamespace, method.methodName)
 
+          methodConfigDetailsPage.isLoaded shouldBe true
```

Comment:
add an additional check here on the imported method name, just to verify that we've imported the correct thing and navigated to the right place?